server/src/inmantals/server.py

```diff
@@ -224,18 +224,20 @@ def create_tmp_project(self) -> str:
 
             raise InvalidExtensionSetup(error_message)
 
-        repos: str = self.handler.repos if self.handler.repos else ""
+        metadata: typing.Mapping[str, object] = {
+            "name": "Temporary project",
+            "description": "Temporary project",
+            "modulepath": "libs",
+            "downloadpath": "libs",
+            "install_mode": install_mode.value,
+        }
+        if self.handler.repos:
+            metadata["repo"] = self.handler.repos
+        logger.debug(
+            "project.yaml created at %s, repos=%s", os.path.join(inmanta_project_dir, "project.yml"), self.handler.repos
```

Comment:
Do you know how it comes that many log messages are only in this file and not in the output?

Reply:
Not sure if this is what you mean but changing the log level to debug in there https://github.com/inmanta/vscode-inmanta/blob/1cbc27efec7136a0c2096372f906a30391c1ee2a/server/src/inmantals/jsonrpc.py#L141 shows these debug logs in the  language server output. (btw This is a "debug tips and tricks" I'm adding to the docs )

Reply:
That's probably it, thanks!